UI/Dialogs/SettingsDialog.resx

```diff
@@ -501,14 +552,71 @@
   <data name="&gt;&gt;mobiFlightTabPage.ZOrder" xml:space="preserve">
     <value>1</value>
   </data>
+  <data name="joystickMidiPanel.Dock" type="System.Windows.Forms.DockStyle, System.Windows.Forms">
+    <value>Fill</value>
+  </data>
+  <data name="joystickMidiPanel.Location" type="System.Drawing.Point, System.Drawing">
+    <value>3, 3</value>
+  </data>
+  <data name="joystickMidiPanel.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>4, 5, 4, 5</value>
+  </data>
+  <data name="joystickMidiPanel.Size" type="System.Drawing.Size, System.Drawing">
+    <value>842, 843</value>
+  </data>
+  <data name="joystickMidiPanel.TabIndex" type="System.Int32, mscorlib">
+    <value>0</value>
+  </data>
+  <data name="&gt;&gt;joystickMidiPanel.Name" xml:space="preserve">
+    <value>joystickMidiPanel</value>
+  </data>
+  <data name="&gt;&gt;joystickMidiPanel.Type" xml:space="preserve">
+    <value>MobiFlight.UI.Panels.Settings.JoystickMidiPanel, MFConnector, Version=10.0.0.0, Culture=neutral, PublicKeyToken=null</value>
+  </data>
+  <data name="&gt;&gt;joystickMidiPanel.Parent" xml:space="preserve">
+    <value>tabPage1</value>
+  </data>
+  <data name="&gt;&gt;joystickMidiPanel.ZOrder" xml:space="preserve">
+    <value>0</value>
+  </data>
+  <data name="tabPage1.Location" type="System.Drawing.Point, System.Drawing">
+    <value>4, 29</value>
+  </data>
+  <data name="tabPage1.Padding" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>3, 3, 3, 3</value>
+  </data>
+  <data name="tabPage1.Size" type="System.Drawing.Size, System.Drawing">
+    <value>848, 849</value>
+  </data>
+  <data name="tabPage1.TabIndex" type="System.Int32, mscorlib">
+    <value>4</value>
+  </data>
+  <data name="tabPage1.Text" xml:space="preserve">
+    <value>Joystick &amp; Midi</value>
```

Comment:
```suggestion
    <value>Joystick &amp; MIDI</value>
```

MIDI is an acronym that's written in all caps (using https://www.midi.org/ as guidance)

Reply:
done